### PR TITLE
FAQ entry on foundation status.

### PR DESCRIPTION
**Pull request by geeknoid**

### CODE REVIEW

File: content/en/faq/general/foundation.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Is Istio a member project of a foundation like the Linux Foundation, CNCF or OASIS?
+weight: 90
+---
+
+No, Istio is an open source project under the [Apache License 2.0]({{< github_blob >}}/LICENSE) that exercises an
+[open governance model](https://github.com/istio/community) with a multi-company [steering committee](https://github.com/istio/community/blob/master/STEERING-COMMITTEE.md),
+[technical oversight committee](https://github.com/istio/community/blob/master/TECH-OVERSIGHT-COMMITTEE.md) and distributed [working groups](https://github.com/istio/community/blob/master/WORKING-GROUPS.md).
+There are no plans to donate Istio to a foundation at this time.
```
Comment (linsun) [inline content/en/faq/general/foundation.md]: can we remove this line about plans?

Comment (jdumars) [inline content/en/faq/general/foundation.md]: This line can be removed.

Comment (geeknoid) [reply content/en/faq/general/foundation.md]: OK, it's gone. You OK with checking this in?

Comment (linsun) [reply content/en/faq/general/foundation.md]: The IBM team would like to have us wait on publishing this FAQ till the senior team have synced and agreed to this, @geeknoid @jdumars

**Pull request closed without merging**